docs/contributing/glossary.md

```diff
@@ -316,6 +326,6 @@ This glossary provides an alphabetized list of technical term definitions used i
 
 ## W
 
-**[Webhook](/docs/alerts/webhook-connections)**. A webhook is an HTTP callback: an HTTP POST that occurs when something happens. Webhook connections allow you to send Sumo Logic alerts to third-party applications that accept incoming webhooks. You can configure a custom webhook, or use preconfigured webhooks for a number of apps, including Slack, Datadog, and so on.
+**[Webhook](/docs/integrations/webhooks/)**. A webhook is an HTTP callback: an HTTP POST that occurs when something happens. Webhook connections allow you to send Sumo Logic alerts to third-party applications that accept incoming webhooks. You can configure a custom webhook, or use preconfigured webhooks for a number of apps, including Slack, Datadog, and so on.
```

Comment:
I would remove this because _webhook_ (and _webhook connection_) is a common industry term.

```suggestion
```